crates/fig_os_shim/src/process_info/pid.rs

```diff
@@ -1,16 +1,34 @@
 use std::path::PathBuf;
 use std::sync::Weak;
-use std::{fmt, str};
+use std::{
```

Comment:
The changes in this diff were produced by executing `cargo +nightly fmt`.
It includes no functional modifications.